tests/unit/UseCases/PushDeletedToCyclopsUseCaseTest.php

```diff
@@ -43,7 +43,7 @@ public function testDeletedPushedToCyclops()
         $list->items = [$createEntity('test@test.com'), $createEntity('test@testtest.com')];
 
         $useCase = new PushDeletedToCyclopsUseCase($service);
-        $useCase->execute($list);
+        $useCase->execute($list, function() {});
```

Comment:
i find it a cleaner/more common pattern to make callbacks option. Just default it to null and wrap the call in an in

Reply:
🎉Updated 🎉